### DISCUSSION TITLE
`That page doesn't exist` error after CLI installtion.

### DISCUSSION THREAD

**Discussion by shree0** [chevereto/chevereto#19] [Q&A]
After the command `app/bin/legacy -C install  -u .... -e ..... -x .... `, the login page shows `That page doesn't exist. The requested page was not found.` error and unable to load the login URL.

**Comment by rodber** [comment]
You require a server with URL rewriting.
https://v4-docs.chevereto.com/application/stack/web-server.html

**Reply by shree0** [reply]
Is there any difference in settings for root installation and directory installation? I am able to install it in the root but the directory through installation not working (getting 404 Not found error).

**Reply by rodber** [reply]
Yes, you need to configure the hostname path.
https://v4-docs.chevereto.com/application/configuration/environment.html#hostname-variables

```
CHEVERETO_HOSTNAME_PATH=/mypath/
```

**Reply by shree0** [reply]
I had set the hostname path in my `.env` file. (CHEVERETO_HOSTNAME_PATH=/mypath/) Now I am getting `This page isn’t working` error. (Unable to get the cause for this)
